### ISSUE TITLE
Cluster split animation is inaccurate

### ISSUE THREAD

**Issue by CostinTanasoiu** [Viglino/OL3-AnimatedCluster#19]
Hi, I've noticed some strange behavior: when zooming in, some clusters will split, but the animation and smaller clusters into which they split are misleading. I have recorded this behavior here: https://www.screencast.com/t/psP4s4bu

When zooming in, the orange cluster with 6 items seems to split into two clusters with 4 items and 1 item respectively. However, what isn't noticeable at first is that one of those 6 items actually gets redistributed into another cluster. If you look at the tooltips in the video, it's the item labeled "32 Grosvenor Gardens, London SW1W0DH". 

Basically, from this: 

![image](https://user-images.githubusercontent.com/2901534/34953687-7f1c30ea-fa15-11e7-9519-a5acdf0e889f.png)

to this: 

![image](https://user-images.githubusercontent.com/2901534/34953658-69bc151c-fa15-11e7-8927-9a96dacc40cf.png)

It would be super useful if the animation showed that item splitting from the big cluster and moving into the other cluster.

I have seen this happen quite often too, with other clusters.
